src/development/python_package_release_procedure.rst

```diff
@@ -0,0 +1,118 @@
+.. _Semver: https://semver.org
+
```

Comment:
Add a page title and move every section down of one level.
This way the left sidebar will just have one entry that is exploded into subsections.